projects/com.oracle.truffle.llvm.nodes.impl/src/com/oracle/truffle/llvm/nodes/impl/base/LLVMContext.java

```diff
@@ -128,7 +129,7 @@ public Source getSourceFile() {
 
     // TODO No static access to this class from LLVMFunction at the moment
 
-    private static LLVMContext lastContext;
+    @CompilationFinal private static LLVMContext lastContext;
```

Comment:
Really this needs to be properly fixed, rather than worked around. We should get on and do that after this is merged.